### PR TITLE
Updated Alazar 9360 notebook to remove legacy loop method

### PR DESCRIPTION
**Pull request by FarBo**
The name of the file is slightly changed (ATS9360 instead of 9360)
@jenshnielsen

### CODE REVIEW

**Review by jenshnielsen — APPROVED** [review]

**Comment by jenshnielsen** [comment]
Did you also want to link to the Controllers in qdev_wrappers ?

**Comment by FarBo** [comment]
I will add that in a commit.

**Comment by FarBo** [comment]
@jenshnielsen 
Could you check the added links to qdev-wrappers?

File: docs/examples/driver_examples/Qcodes example with Alazar ATS9360.ipynb [hunk 1]
```diff
@@ -997,6 +997,20 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Looking for extra functionalities?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```
Comment (jenshnielsen) [inline docs/examples/driver_examples/Qcodes example with Alazar ATS9360.ipynb:1011]: I would like to write something like.

It is expected that the user will implement acquisition to fit their needs one can visit this link for some examples of other controllers ....

**Review by jenshnielsen — APPROVED** [review]

**Pull request merged**